tac/guidelines/project-consistency-guidelines.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Project Consistency Guidelines
+parent: Guidelines
+grand_parent: LF Decentralized Trust TAC
+nav_order: 7
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+This document describes various considerations projects should consider in order to maintain consistency across different projects.
+
+There are several reasons why consistency across projects is beneficial.
+
+## Benefits to the LFDT
+
+The LFDT manages a growing number of projects and labs and must review and discuss all of the projects on a regular basis. This process of governance is much easier when projects are managed and run in a consistent way.
+
+## Benefits to project maintainers
```

Comment:
A suggested benefit - projects will be of highest standards, structured to follow best practices and guidance in LFDT without having to reinvent wheels.